core/src/test/java/com/mapzen/android/graphics/MapzenMapTest.java

```diff
@@ -293,10 +293,10 @@
     verify(overlayManager).onFling(1f, 2f, 3f, 4f);
   }
 
-  @Test public void setRotateResponder_shouldInvokeMapController() {
+  @Test public void setRotateResponder_shouldNotInvokeMapController() {
```

Comment:
Can you add these two tests into `MapzenMapTest`, update `TestRotateResponder`, and then run `./gradlew testDebug` to make sure that the tests pass? Thank you!